src/schemas/json/io-package.json

```diff
@@ -0,0 +1,504 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "multilingual": {
+      "type": "object",
+      "required": ["en"],
+      "patternProperties": {
+        "en|de|ru|pt|nl|fr|it|es|pl|zh-cn": {
+          "type": "string"
+        }
+      },
+      "additionalProperties": false
+    }
+  },
+  "type": "object",
+  "required": ["common"],
+  "additionalProperties": false,
+  "properties": {
+    "$schema": {
+      "description": "JSON schema for validation",
+      "type": "string"
+    },
+    "common": {
+      "required": ["name", "version", "platform", "titleLang", "news", "desc", "mode"],
+      "additionalProperties": false,
+      "description": "Contains all common instance attributes",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name of the adapter without ioBroker",
+          "type": "string",
+          "pattern": "(?i)^(?!iobroker)"
```

Comment:
The build won't work without force` flag, because it says that it isn't a valid regex while it is. The regex is also correctly validated using the schema in Webstorm IDE. Making this part case insensitive would also be possible, however for the other case, it's not that nice.

I am looking forward to your feedback on this.